css/styles.css

```diff
@@ -264,14 +337,88 @@ textarea::placeholder {
   color: #b3bac5;
 }
 
-.contact-form input.button {
-  align-self: start;
-}
-
 .heading-color { color: #172b4d; }
 
 .project-details-color { color: #7a869a; }
 
-.dark-purple { color: #6070ff; }
-
 .blank-space { height: 50px; }
+
+@media (min-width: 800px) {
+  .card {
```

Comment:
- [ ] I must admit that your project looks amazing and professional 🔥 🔥, keep rocking!!!

- [ ] Nice one implementing a media query for the desktop breakpoint. However, the required value for `min-width` property should be in range of `992px - 1100px`. This means that you can pick any value between `992px` and `1100px `as the `800px` you have is too small for a desktop breakpoint.

- [ ] I can see you used only flexbox to layout elements on the page. Have you thought of using the [grid system](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids) to layout some of the elements? It looks pretty cool to use both `flexbox` and `grid` to layout elements so that you can have the knowledge of using both comfortably when displaying elements on the page. Please let's use `grid` to display at least one element on the page 👍

Reply:
Hi, Thanks for the review.

If you notice, I have used Grid to display Works Section since it was required. 

![image](https://user-images.githubusercontent.com/73714615/149123664-d29e87b0-f906-47b3-b188-6455a12239dd.png)

Other than that I don't think, it could be used anywhere else without introducing extra complexities.

Reply:
Other than that, I have changed the desktop break-point.